src/CarbonAware.WebApi/test/integrationTests/CarbonAwareControllerTests.cs

```diff
@@ -226,18 +229,20 @@ public async Task EmissionsForecastsBatch_SupportedDataSources_ReturnsOk(DateTim
         }
     }
 
-    [TestCase("2022-1-1T04:05:06Z", "2022-1-2T04:05:06Z", "eastus", TestName = "EmissionsActual expects OK for eastus")]
-    [TestCase("2021-12-25", "2021-12-26", "westus", TestName = "EmissionsActual expects OK for westus")]
-    public async Task EmissionsActual_ReturnsOk(DateTimeOffset start, DateTimeOffset end, string location)
+    [TestCase("2022-1-1T04:05:06Z", "2022-1-2T04:05:06Z", "eastus", TestName = "EmissionsMarginalCarbonIntensity expects OK for eastus")]
```

Comment:
@juzuluag reopened this because it felt like it was closed by accident, given that you took the suggestion on the testcase below it and it would be good for the test cases to match.